es-core/src/ThemeData.h

```diff
@@ -73,6 +73,19 @@ ThemeException& operator<<(ThemeException& e, T appendMsg)
 	return e;
 }
 
+class ThemeExtras : public GuiComponent
```

Comment:
The `ThemeExtras` class was removed in the time since this grid view implementation was originally done.  This change needs to be reverted.